docs/en/contributing/install.md

```diff
@@ -8,33 +8,28 @@ sidebar_label: Install
 
 ## Prerequisites
 
-Before setting up FAST-DNA, install Git and [yarn](https://yarnpkg.com/en/docs/install).
+Before setting up FAST-DNA, install Git and <a href="https://yarnpkg.com/en/docs/install" target="_blank">Yarn</a>.
 
 ## Setup the source repository
 
-Clone the repository, `cd` into the project, install [Lerna](https://github.com/lerna/lerna) and install dependencies:
+Clone the repository, `cd` into the project, install <a href="https://github.com/lerna/lerna" target="_blank">Lerna</a> and install dependencies:
 
 ```shell
 git clone https://github.com/Microsoft/fast-dna.git
 cd fast-dna
 yarn global add lerna
 ```
 
-Install all dependencies:
+Install and symbolically link dependencies across all workspaces:
 
 ```shell
 yarn install
 ```
 
-Prepare the workspaces:
-
-```shell
-lerna run prepare
-```
-
+- `lerna run prepare` to symbolically link dependencies
```

Comment:
`lerna run prepare` does not symbolically link dependencies, it builds the packages